tools/sylph/sylph_to_taxprof.py

```diff
@@ -0,0 +1,101 @@
+import pandas as pd
```

Comment:
It seems that both Python scripts come from https://github.com/bluenote-1577/sylph-utils.
There is no LICENSE in the repository, so you are not allowed to take the scripts and add them here.
It might be better to find a way (maybe talking with authors) so they make the scripts available via conda. That would avoid duplication

Reply:
I open 2 issues in sylph-utils:
- LICENSE: https://github.com/bluenote-1577/sylph-utils/issues/1
- Packaging: https://github.com/bluenote-1577/sylph-utils/issues/2